hr_holidays_previous_type/README.rst

```diff
@@ -0,0 +1,49 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+=========================
+HR holidays previous type
+=========================
+
+This module was written to allow to define a previous type on holidays type.
+if there are any valid days in the previous type, a warning is raised.
+
```

Comment:
I don't understand this. Could you explain in a different way, or maybe give an example?